src/advanced/time.md

```diff
@@ -268,6 +268,10 @@ the tight integration with consensus. This approach is more flexible and
 manageable, and could be generalized to the agreement between arbitrary
 collectively trusted entities, which may behave maliciously.
 
+## Other languages support
```

Comment:
I think it must be at the top (not as a separate section, but more like a declaration of supported languages) — no one scrolls that much down.